Automattic/jetpack
# CRM: Fix learn menu buttons overlapping

## Proposed changes:

This PR fixes an alignment issue in the CRM Learn menu.
It is a simple fix wrapping its contents when it is not big enough to fit all buttons.
This fix looks good enough for the moment, a more elaborate fix should be done when fixing the CRM for mobile (small screen) devices.

### Other information:


## Jetpack product discussion
https://github.com/Automattic/zero-bs-crm/issues/3147

## Does this pull request change what data or activity we track or use?
No.

## Testing instructions:

* Open the Edit Invoice page
* Make the window small enough to make the buttons overlap. (This should happen near 1519px wide)

In `trunk` the buttons will overlap.
![image](https://github.com/Automattic/jetpack/assets/37049295/6c172fd9-4a17-431c-9c16-b9ca073347e1)


In `fix/crm/3147-learn-menu-buttons-overlapping` the buttons will wrap.
![image](https://github.com/Automattic/jetpack/assets/37049295/6c835dfd-5e88-4064-a793-023d4d41947f)
